public/themes/pterodactyl/js/frontend/nodes/allocation.js

```diff
@@ -0,0 +1,115 @@
+class Allocation {
```

Comment:
Can't use any of this unless you add it to babel to compile into ES2015. :(

Reply:
Should I do that or just inline the code (sans class) in `allocation.blade.php`?

Reply:
Can you just write it using ES5?

Reply:
Should be done.